### DISCUSSION TITLE
Calculation of AUCR and CmaxR for DDI simulations

### DISCUSSION THREAD

**Discussion by PavelBal** [Open-Systems-Pharmacology/Forum#1138] [Questions & Problems]
Could you please explain how exactly the calculations of the AUC and Cmax ratios in the PK-Analysis tab are performed? I am especially interested in what steps are performed for the Intraconazole project, where the CYP3A4 metabolization chain is

Itraconazole -> Hydroxy-Itraconazole -> Keto-Itraconazole -> N-desalkyl-Itraconazol

and all 4 compounds are inhibitors of CYP3A4.

**Comment by msevestre** [comment]
The calculation is performed for one simulation and one compound,
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Core/Services/GlobalPKAnalysisTask.cs#L276

even though the knowledge of the compound name is not very useful because we are de-facto turning off ALL inhibitions

https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/617264dee8e040bb5f71d46f74efc891641eea6c/src/PKSim.Core/Model/SimulationFactory.cs#L208

This is the place here
```
_interactionTask.AllInteractionContainers(ddiRatioSimulation)
            .SelectMany(c => c.AllParameters())
            .Each(p => p.Value = disabledDDIValueFor(p.Name));
```
In short, we create a clone of the original simulation, we turn off all interaction processes (ALL of them) and we simulate.
So in your case, inhibition is switched off for all Itra* compounds of the model

Does it help?

**Reply by msevestre** [reply]
This also means we could calculate the AUCR etc.. for all compounds at once I believe. Bioavailability is a different beast because we are changing the protocol of one single compound

**Comment by PavelBal** [comment]
> Does it help?

Yepp, definitely. I am not sure yet how to interpret this, but at least the technical part is clear.

**Comment by StephanSchaller** [comment]
Hm, might this not be an issue if we have a victim compound with e.g. self-inhibition:
let's assume itraconazole was also evaluated with another CYP3A4 inhibitor ("X") to investigate the effect of this inhibitor X on itraconazole PK, would we ever get the right AUCR for itraconazole for this case (as we would not want to switch off the self-inhibition of itraconazole (metabolites) for this case but only the one from inhibitor X.

@sfrechen ?

**Reply by StephanSchaller** [reply]
maybe we need a prompt where we can then select the DDIs to switch off (same for Bioavailability where we then select the administrations to switch from po to iv)?

**Comment by msevestre** [comment]
For bioavailability this is clear. You already click on a given drug. For DDI, this could be a good option if required...

**Comment by msevestre** [comment]
@StephanSchaller We do not model auto-inhibition explicitly in PK-Sim. I thought this was already included in the Ki/Km values. So how would this solve the problem at hand?

**Reply by StephanSchaller** [reply]
Not auto-inhibition, but inhibition by a metabolite. If you take, e.g., itraconazole, its own 3A4 metabolism is inhibited by 2-3 of its own metabolites (which are modelled).
So if you combine it in a DDI with midazolam, calculating the DDI via this button would also switch of the interaction of these metabolites, for the calculation of the nonperturbed scenario, which is not what you would want. You would only want the DDI on midazolam to be switched off, right?

**Reply by msevestre** [reply]
Yep agreed
